CMakeLists.txt

```diff
@@ -226,6 +277,25 @@ set(GRPC_GENERATE_CPP_APPEND_PATH OFF)
 # By default, compile with all warning flags
 add_compile_options(-Wall -Wextra)
 
+# Add extra configuration files to run additional
```

Comment:
Line-break looks very early.  Similarly for comment below.

Reply:
Should be fixed in https://github.com/clearmatics/zeth/pull/208/commits/2be3993bf0895c2afc8b957158c31b94f61ff2e6